pyproject.toml

```diff
@@ -42,6 +42,7 @@ tune = "torchtune._cli.tune:main"
 [project.optional-dependencies]
 dev = [
     "bitsandbytes>=0.43.0",
+    "comet_ml>=3.44.2",
```

Comment:
to confirm, this is the correct version for the comet v2 API?

Reply:
Yes, this version contains our latest change and fixes related to `comet_ml.start` and relevant to this integration